### DISCUSSION TITLE
Update 0.14.3 - 1.x

### DISCUSSION THREAD

**Discussion by Helmut2006** [Louisvdw/dbus-serialbattery#722] [General]
Hello,
What is the correct procedure? I'm running 0.14.3 and want to update to 1.x. First uninstall 10.14.3 completely or simply install 1.x. Point by point instructions would help me a lot. Thanks very much

**Comment by mr-manuel** [comment]
Just install

**Comment by packrat1969** [comment]
mr. manuel,

If I "just install" the current 1.xx version, won't the older 0.14.3 version I currently have installed interfere with the newly installed version?

Is there any way to make the name of the battery/BMS "stick" following a system reboot?

Are there any plans to coordinate with kwindrem to use his "Package Manger" arrangement with "Active" and "Inactive" packages to make the whole installation/update process a bit more painless and a bit less doubtful?  It's getting the proper files in the proper locations and removing any unnecessary/obsolete files that I am concerned with.  I am not above editing a configuration file here and there once things are in place, but I am NOT a Linux expert. 

packrat

**Reply by LarsAndersenDenmark** [reply]
What did you do?
Installed the new without removing old dbus driver?

**Reply by mr-manuel** [reply]
The install process always cleans up the current version and then installs the new one. So no interference, if you update. If you downgrade, then it's better if you uninstall the driver first.

The name stick with the battery since a long time now, just update to the latest version.

No plans to coordinate with Setup Helper/Package Manager, but maybe there is something other coming.

No need to be a Linux expert. Just follow step by step the instructions.

**Reply by LarsAndersenDenmark** [reply]
Great, thanks for fast response.
Do it also transfer settings or is this to be done. Guess its a clean install due to the change of having persistent settings in the new driver.
Do you know which Venus OS for raspi 2 is the latest stable working revision that works with the new dbus-driver?

**Reply by mr-manuel** [reply]
Settings are preserved only, if your system is already `>=1.0.0`.

The latest stable Venus OS is always working with the latest driver version. If it's not compatible the driver will tell you on install or in the logs, if already installed.

**Reply by LarsAndersenDenmark** [reply]
Ok, just heard that the new 3.5 did not work with dbus-serialbattery and want to be sure to not stall my system completely.
Will do a copy of the SD card on my macbook as backup and try updating to newest VenusOS on the raspi 2 and try install newest dbus-driver.

**Reply by mr-manuel** [reply]
@LarsAndersenDenmark who said that? Please share a link, if possible. I just tested it on three different systems and the installation worked on all.
